buildenv/jenkins/JenkinsfileBase

```diff
@@ -112,7 +112,7 @@ def setupEnv() {
 
 	env.LIGHT_WEIGHT_CHECKOUT = (params.LIGHT_WEIGHT_CHECKOUT == false) ? params.LIGHT_WEIGHT_CHECKOUT : true
 	env.GENERATE_JOBS = params.GENERATE_JOBS ?: false
-	env.isSVTTestRepo=false
+	env.IS_SVT_TESTREPO = ""
```

Comment:
I do not think this line is needed.

Reply:
ok,I was providing this line to check this variable is set or not set. Will try removing this line and test.